chat.js

```diff
@@ -389,6 +397,9 @@ class CommandContext {
 		if (room.filterCaps && user.name.match(/[A-Z\s]{6,}/)) {
 			return this.errorReply(`Your username contains too many capital letters, which this room doesn't allow.`);
 		}
+        if (room.filterEmojis && emojiRegex.exec(user.name)) {
```

Comment:
What emoji definition should be used then? The external dependency I use at the moment makes use of [unicode technical report 51](https://github.com/mathiasbynens/unicode-tr51)